doc/index.rst

```diff
@@ -40,9 +40,9 @@ Catalyst
     :description: Get started using Catalyst with PennyLane
 
 .. index-card::
-    :name: API
-    :link: code/__init__.html
-    :description: Explore the Catalyst API
+    :name: Source
```

Comment:
Minor suggestion @dime10, but I would either change this to 'GitHub' or 'Source Code' or 'Code'.

Source (to me) can be a little ambiguous. I think 'GitHub' may be best, to indicate it is an external link

Reply:
```suggestion
    :name: GitHub
```